plugins/reporters/web-app-template/src/components/VulnerabilitiesTable.jsx

```diff
@@ -23,15 +23,10 @@ import {
 } from 'react';
 
 import {
-    CloseCircleOutlined,
-    ExclamationCircleOutlined,
     FileAddOutlined,
     FileExcelOutlined,
     FileProtectOutlined,
-    InfoCircleOutlined,
-    IssuesCloseOutlined,
-    QuestionCircleOutlined,
-    WarningOutlined
+    IssuesCloseOutlined
```

Comment:
Wrong link in commit message, should be https://github.com/oss-review-toolkit/ort/issues/8807.